Package.swift

```diff
@@ -17,7 +17,7 @@ let package = Package(
             targets: ["MapboxCoreMapsWrapper"]),
     ],
     dependencies: [
-        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("10.0.0-beta.12")),
+        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("10.0.2")),
```

Comment:
Should we be switching to "up to next latest" semantics now?

Reply:
cc @brunoabinader @DorkMatter

Reply:
:+1:

Reply:
We'll want to do this in the podspec as well.

To achieve equivalent semantics in the podspec and Package.swift, you use slightly different ways of specifying the dependency:

In the podspec, don't specify the patch version, otherwise, the `~>` operator means "up to next minor":

`'~> 10.0'`

In Package.swift, you can specify the patch version:

`.package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", from: "10.0.2"),`